<h1 class="text-3xl md:text-4xl font-bold text-blue-700 mb-6">Research Projects</h1>

<figure class="bg-light bg-gray-200 p-4 rounded-md">
  <blockquote class="blockquote">
    <p class="text-lg italic">“<em>If you torture the data long enough, Nature will confess...</em>”</p>
  </blockquote>
  <figcaption class="blockquote-footer text-gray-600">
    Ronald Coase
  </figcaption>
</figure>

<div class="card p-3 my-3 bg-light bg-gray-200 border-dark border rounded-md">
  <h3 id='theme-1-genome-scale-bigdata-structural-bioinformatics' class="text-2xl font-bold">THEME-1. Genome-scale (bigdata) structural
    bioinformatics</h3>
  <p class="text-gray-700">We develop and apply advanced computational methodologies in bioinformatics of proteins and RNAs. These methods
    combine novel and existing algorithms drawn from machine learning, statistics, and bigdata analytics. Our projects
    range from analysis and prediction of bigdata in the order of 10 million proteins/RNAs and several thousands of
    organisms, through analysis of individual organisms (thousands of proteins/RNAs), disease models, to analysis of
    individual proteins/RNAs. Our research projects concern - Prediction and modeling of structure and function proteins
    and RNAs - Discovery and characterization of sequence-structure/disorder-function relationships in proteins -
    Binding of small ligands (including drugs), peptides, RNAs and DNA to proteins - Prediction and functional
    characterization of intrinsic disorder in proteins </p>
</div>

<div class="card p-3 my-3 bg-light bg-gray-200 border-dark border rounded-md">
  <h3 id='theme-2-translational-analysis-of-complex-biomedical-data' class="text-2xl font-bold">THEME-2. Translational analysis of complex biomedical
    data.</h3>
  <p class="text-gray-700">We develop advanced translational systems that are used to convert complex (multi-system, multi-source and
    multi-format) clinical data into diagnostically- and prognostically-relevant observations. We collaborate with
    computer scientists, bioinformaticians, biochemists, structural biologists, biophysicists and clinicians to analyze
    a variety of such datasets that are relevant to several human diseases including cystic fibrosis, arthritis, and
    heart failure. </p>
</div>
<p class="text-gray-700">A sample of our projects follows.</p>
<h4 class="mt-3 text-xl font-bold" id='high-throughput-prediction-of-binding-sites-and-regions-in-proteins-theme-1'>High-throughput prediction of
  binding sites and regions in proteins (THEME-1)</h4>

<p class="text-gray-700">Prediction of binding sites has applications in rational drug design and in the discovery of protein-protein and
  protein-ligand interactions. We developed several machine learning methods that predict binding residues in the
  protein sequence and binding sites in the protein structures. Our efforts concentrate on predictions for:</p>
<ul class="list-disc list-inside text-gray-700">
  <li>specific ligands. For instance, we developed RBRpred method [<em>Curr Prot Pept Sci</em> 11(7):609-28, 2010]
    that predicts RNA-binding residues, NsitePred [<em>Bioinformatics</em> 28(3):331-41, 2012] that finds
    nucleotide-binding residues, CRpred [<em>Bioinformatics</em> 24(20):2329-38, 2008] that predicts catalytic
    residues in protein chains.</li>
  <li>for specific proteins. For instance, we proposed a novel method for predictions of hot spots in tubulin isotypes
    [<em>J Mol Graph Model</em> 27(4):497-505, 2008].</li>
  <li>for blind (without the knowledge of the ligand) prediction of binding sites on a proteome scale [<em>PLoS
    ONE</em> 4(2):e4473, 2009; <em>Structure</em> 20:1815-22, 2012].</li>

</ul>
<p class="text-gray-700">We performed a comprehensive critical evaluations of in-silico predictors of the binding sites for biologically
  relevant organic compounds. [<em>Structure</em> 19:613-21, 2011] and for sequence-based prediction of DNA- and
  RNA-binding [<em>Brief Bioinformatics</em> DOI 10.1093/bib/bbv023, 2015]. This former contribution highlights the
  current developments in this area and positions our group at the forefront of this field. We are in the possession
  of unique and invaluable insights into building of the next generation predictors and we authored the most
  comprehensive benchmark dataset and results. We currently focus on the inverse binding site predictions, where
  knowledge of one or a few protein-ligand complexes is used to find other binding partners for this ligand on a
  proteomic (whole organism) scale. This work finds applications in discovery, characterization, and mitigation of
  side effects of clinically used and novel small molecule-based drugs and in drug repurposing. We published a
  cutting-edge inverse binding site predictor, ILbind, which provides favorable predictive performance on a large and
  diverse set of 35 compounds [<em>Structure</em> 20:1815-22, 2012] and comprehensively characterized novel
  interactions between human proteins and a widely used immunosuppressant Cyclosporin A <em>[Bioinformatics</em>
  30(24):3561-66, 2014].</p>


<p class="text-center"><img class="img-fluid p-3 mx-auto" src="images/projectsInverseDocking.jpg" referrerpolicy="no-referrer"
                            alt="inverse binding site prediction"></p>
<p class="text-center"><img class="img-fluid p-3 mx-auto" src="images/projectsNucleotides.jpg" referrerpolicy="no-referrer"
                            alt="binding site prediction"></p>
<h4 class="mt-3 text-xl font-bold" id='characterization-and-prediction-of-disorder-in-proteins-theme-1'>Characterization and prediction of disorder in
  proteins (THEME-1)</h4>

<p class="text-gray-700">For years, scientists were convinced that proteins must fold into precise, rigid molecules to function correctly.
  This view is changing now. The intrinsically disordered proteins (IDPs) have at least some disordered (also
  called unfolded or highly flexible) regions and many of them carry out their function without ever fully folding
  into a rigid molecule. This flexibility is crucial for signal transduction, regulation of cell division,
  transcription, translation, phosphorylation, and many other cellular functions. The prevalence of disorder was
  also shown in human diseases including cancers, cardiovascular, neurodegenerative, and genetic diseases.
  Experimental annotations of IDPs are time consuming and difficult and thus computational methods that predict
  disorder from sequences have emerged as a viable alternative to investigate the disorder. These methods find
  numerous applications in functional and structural proteomics.</p>
<p class="text-gray-700">We have a history of excellence in this area. Our machine learning predictor, MFDp [<em>Bioinformatics</em>
  26(18):i489-96, 2010], recently secured top three finish among 28 participants in the disorder prediction at the
  10th worldwide CASP10 experiment [<em>Proteins</em> 82(Suppl 2):127-37, 2014]. We also developed
  first-of-its-kind predictor of disorder content, DisCon [<em>BMC Bioinformatics</em> 12:245, 2011], one of the
  most accurate methods for the prediction of disordered protein-binding regions, MoRFpred
  [<em>Bioinformatics</em> 28(12):i75-i83, 2012], high-throughput (genomic scale) predictor RAPID [<em>BBA Prot
    Proteomics</em> 1834(8):1671-80, 2013], the first-of-its-kind predictor of multiple disorder driven
  functions, disoRDPbind [<em>Nucleic Acids Res</em> 43(18):e121, 2015], that covers protein-, RNA- and
  DNA-binding, and we co-developed the repository of putative disorder [<em>Nucleic Acids Res</em> 41:D508-16,
  2012]. Our methods are available as convenient webservers that enjoy a wide-spread use; we have processed
  requests from thousands of unique users from 80+ countries on all continents.</p>
<p class="text-gray-700">We also published one of the most comprehensive comparative reviews on the prediction of intrinsic disorder
  [<em>Curr Prot Pept Sci</em> 13:6-18, 2012] and performed characterization of the functional roles that the
  disorder plays. We just completed several ambitious, collaborative projects that analyzed the role of disorder
  in the proteome of the HIV-1 virus [<em>Cell Mol Life Sci</em> 69:1211-59, 2012], hepatitis C virus [<em>Mol
    BioSystems</em> 10:1345-1363, 2014] and dengue virus [<em>FEBS J</em> 282(17):3368-3394, 2015], in the
  nucleosome [<em>Mol BioSystems</em> 8:1886-901, 2012], in the programmed cell death peocess [<em>Cell Death and
    Differ</em> 20:1257-67, 2013], and in the ribosomal proteins [<em>Cell Mol Life Sci</em> 71(8):1477-1504,
  2013]. As a recent highlight, we recently contributed to two landmark reviews on the role of disorder in protein
  complexes [<em>Chemical Rev</em> 114 (13):6806-43, 2014] and in viruses [<em>Chemical Rev</em> 114(13):6880-911,
  2014]. We also published arguably the most comprehensive to date characterization of the abundance and
  functional roles of intrinsic disorder over all known domains of life [<em>Cell Mol Life Sci</em> 72(1):137-151,
  2015] and on the abundance of MoRF regions [<em>Mol BioSystems</em> DOI: 10.1039/C5MB00640F, 2015].</p>


<p class="text-center"><img class="img-fluid p-3 mx-auto" src="images/projectsMoRFpred.jpg" referrerpolicy="no-referrer"
                            alt="MoRFpred"></p>
<h4 class="mt-3 text-xl font-bold" id='high-throughput-characterization-of-micrornas-and-their-targets-theme-1'>High-throughput characterization of
  microRNAs and their targets (THEME-1)</h4>

<p class="text-gray-700">MicroRNAs (miRNAs) are abundant and short endogenous noncoding RNAs made of 19-23 nucleotides that bind to target
  mRNAs, typically resulting in their degradation and translational repression. The fine-tuning of gene regulation
  in biological processes and disease pathways by these small RNAs recently attracted significant attention. We
  have developed advanced computational platform to characterize miRNA in large-scale deep-sequencing data and
  built novel methods for high-throughput machine learning-driven prediction of targets of known and novel miRNAs.
</body>
